docs/html/user_guide.rst

```diff
@@ -772,6 +772,172 @@ archives are built with identical packages.
+| ``==``     | Exactly the specified version                                                                                                                                         | **``==3.1``:** only version ``3.1``                                                                                                                           |
++------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``!=``     | Any version not equal to the specified version                                                                                                                        | **``!=3.1``:** any version other than ``3.1``                                                                                                                 |
++------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``~=``     | Any compatible release. Compatible releases are releases that are within the same major or minor version, assuming the package author is using semantic versioning.   | **``~=3.1``:** version ``3.1`` or later, but not version ``4.0`` or later. **``~=3.1.2``:** version ``3.1.2`` or later, but not version ``3.2.0`` or later.   |
++------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``*``      | Can be used at the end of a version number to represent "all"                                                                                                         | **``== 3.1.*``:** any version that starts with ``3.1``. Equivalent to ``~=3.1.0``.                                                                            |
++------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
Could you please wrap the table to fit 80 columns?  I believe that it is possible to do so under reST, otherwise [definition list](https://docutils.sourceforge.io/docs/user/rst/quickref.html#definition-lists) may be used.

Reply:
Using a definition list is a great idea, IMO. :)

Reply:
Using a definition list is great! And I'm grateful @McSinyx for the reStructuredText tips and tricks you have offered in the code review suggestions!

However: I'd like us to first review the CONTENT that @nlhkabu has written, and the general structure of the document, and help her revise that. And only after that will it make sense to optimize for character limits, fix the less crucial syntax things, etc. [As Sage Sharp puts it in "The Gentle Art of Patch Review"](https://sage.thesharps.us/2014/09/01/the-gentle-art-of-patch-review/), review to determine "Is the contribution polished?" should come AFTER review to determine "Is the idea behind the contribution sound?" and "Is the contribution architected correctly?"

Does that make sense?

Reply:
Yes, that makes sense and I'm sorry for, as usual, bikeshedding too soon during reviewing process.